tests/system/Database/Live/MySQLi/FoundRowsTest.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Database\Live\MySQLi;
+
+use CodeIgniter\Test\CIUnitTestCase;
+use CodeIgniter\Test\DatabaseTestTrait;
+use Config\Database;
+use PHPUnit\Framework\Attributes\Group;
+use Tests\Support\Database\Seeds\CITestSeeder;
+
+/**
+ * @internal
+ */
+#[Group('DatabaseLive')]
+final class FoundRowsTest extends CIUnitTestCase
+{
+    use DatabaseTestTrait;
+
+    /**
+     * @var array<string,mixed>
+     */
+    private $tests;
+
+    protected $refresh = true;
+    protected $seed    = CITestSeeder::class;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $config = config('Database');
+
+        $this->tests = $config->tests;
+
+        if ($this->tests['DBDriver'] !== 'MySQLi') {
+            $this->markTestSkipped('Only MySQLi can complete this test.');
+        }
```

Comment:
Move `parent::setUp();` to the bottom.

```suggestion
        $config = config('Database');

        $this->tests = $config->tests;

        if ($this->tests['DBDriver'] !== 'MySQLi') {
            $this->markTestSkipped('Only MySQLi can complete this test.');
        }
        
        parent::setUp();
```